### PR TITLE
chores: build muti-arch image with github action

### PR DESCRIPTION
**Pull request by dongjiang1989**
Build muti-arch images for `ghcr.io` with github actions.

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          -
+            name: open-local
+            image: alibaba/open-local
+            file: ./Dockerfile.muti
+            platforms: linux/amd64,linux/arm64
+
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v3
```
Comment (dongjiang1989) [inline .github/workflows/build-release.yml:35]:
```suggestion
        uses: actions/checkout@v3
     - 
        name: Set up Go
        uses: actions/setup-go@v2.1.4
        with:
          go-version: 1.18
```

File: Dockerfile.muti [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18 AS builder
+
+WORKDIR /go/src/github.com/alibaba/open-local
+COPY . .
+RUN make build && chmod +x bin/open-local
+
+FROM --platform=linux/amd64 openlocal/open-local-base:latest
```
Comment (dongjiang1989) [inline Dockerfile.muti:7]:
```suggestion
FROM openlocal/open-local-base:latest
```

**Comment by dongjiang1989** [comment]
@peter-wangxu  PTAL

File: Dockerfile.muti [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18 AS builder
+
+WORKDIR /go/src/github.com/alibaba/open-local
+COPY . .
+RUN make build && chmod +x bin/open-local
+
+FROM openlocal/open-local-base:latest
+LABEL maintainers="Alibaba Cloud Authors"
+LABEL description="open-local is a local disk management system"
+COPY --from=builder /go/src/github.com/alibaba/open-local/bin/open-local /bin/open-local
```
Comment (peter-wangxu) [inline Dockerfile.muti:10]: Looks restic tools is not included ?

Comment (dongjiang1989) [reply Dockerfile.muti:10]: Hmmm... I miss it.  Fixed, Please re-check.

**Review by peter-wangxu — APPROVED** [review]
LGTM

**Pull request merged**